content/get-started/05_persisting_data.md

```diff
@@ -34,24 +30,43 @@ What you'll see is that the files created in one container aren't available in a
     commands (why you have the `&&`). The first portion picks a single random number and writes
     it to `/data.txt`. The second command is simply watching a file to keep the container running.
 
-2. Validate that you can see the output by accessing the terminal in the container. To do so, you can use the CLI or Docker Desktop's graphical interface.
+2. Validate that you can see the output by accessing the terminal in the container. To do so, you can use the CLI or Docker Desktop's graphical interface. For the CLI, you can use one of the built-in interfaces, including the terminal on Mac or Linux, or PowerShell or Command Prompt on Windows. Additionally on Windows, you can use Git Bash.
```

Comment:
This is so long-winded now. For a getting started guide, it doesn't feel right to go into this level of detail. We could change the example here to avoid it. For example:

```console
$ docker run -it --name=mytest alpine
/ # echo "hello" > greeting.txt
/ # exit
$ docker container start -ia mytest
/ # cat greeting.txt
hello
/ # exit
$ docker run alpine cat greeting.txt
cat: can't open 'greeting.txt': No such file or directory
```

Reply:
Updated. Removed the tabs because there are no syntax differences in the CLI for that example and it's not possible in the GUI.